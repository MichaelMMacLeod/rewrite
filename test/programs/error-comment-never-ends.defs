/* this comment never ends...

it just

        keeps





                going




                            on!