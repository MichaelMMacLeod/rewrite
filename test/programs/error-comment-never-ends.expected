error[E001]: bad syntax
./test/programs/error-comment-never-ends.defs:16:32
    |
 16 |                             on!
    |                                ^ unexpected end of input
    |                                  expecting "*/" or "/*"
